src/main/java/org/folio/event/UserConsortiaEventType.java

```diff
@@ -0,0 +1,15 @@
+package org.folio.event;
+
+public enum UserConsortiaEventType {
```

Comment:
Please remove Consortia from each java class name, not only mod-consortia can consume these events in future, so lets make names more generic